src/components/Library/Library.jsx

```diff
@@ -1,40 +1,41 @@
-import React, { useEffect, useState } from 'react';
-import PropTypes from 'prop-types';
-import { orderBy } from 'lodash';
+import AddIcon from '@mui/icons-material/Add';
+import DeleteIcon from '@mui/icons-material/Delete';
+import StarIcon from '@mui/icons-material/Star';
 import Box from '@mui/material/Box';
+import CircularProgress from '@mui/material/CircularProgress';
+import IconButton from '@mui/material/IconButton';
+import Paper from '@mui/material/Paper';
 import Table from '@mui/material/Table';
 import TableBody from '@mui/material/TableBody';
-import CircularProgress from '@mui/material/CircularProgress';
 import TableCell from '@mui/material/TableCell';
 import TableContainer from '@mui/material/TableContainer';
 import TableHead from '@mui/material/TableHead';
 import TableRow from '@mui/material/TableRow';
-import StarIcon from '@mui/icons-material/Star';
-import Paper from '@mui/material/Paper';
-import IconButton from '@mui/material/IconButton';
 import TableSortLabel from '@mui/material/TableSortLabel';
 import Typography from '@mui/material/Typography';
-import AddIcon from '@mui/icons-material/Add';
-import DeleteIcon from '@mui/icons-material/Delete';
+import { orderBy } from 'lodash';
+import PropTypes from 'prop-types';
+import React, { useMemo, useState } from 'react';
 // import RemoveIcon from '@mui/icons-material/Remove';
+import EditIcon from '@mui/icons-material/Edit';
+import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
+import SentimentVeryDissatisfiedIcon from '@mui/icons-material/SentimentVeryDissatisfied';
+import ShareIcon from '@mui/icons-material/Share';
 import Dialog from '@mui/material/Dialog';
 import DialogActions from '@mui/material/DialogActions';
 import DialogContent from '@mui/material/DialogContent';
 import DialogContentText from '@mui/material/DialogContentText';
 import DialogTitle from '@mui/material/DialogTitle';
-import MoreHorizIcon from '@mui/icons-material/MoreHoriz';
-import EditIcon from '@mui/icons-material/Edit';
-import ShareIcon from '@mui/icons-material/Share';
 /* import { TableVirtuoso } from 'react-virtuoso'; */
 import './Library.css';
 import Classes from './Library.module.css';
 // import rows from './data.json';
-import { Button, Modal, TextField } from '@mui/material';
 import CloseIcon from '@mui/icons-material/Close';
+import { Button, Modal, TextField } from '@mui/material';
 import Snackbar from '@mui/material/Snackbar';
+import axios from 'axios';
 import ISBN from 'isbn-validate';
 import { Rating } from 'react-simple-star-rating';
-import axios from 'axios';
 
 // function createData(name, calories, fat, carbs, protein) {
 // 	return { name, calories, fat, carbs, protein };
```

Comment:
I think I we should incorporate a //TODO in comments because we have mostly TODO issues left.

Unless we switch to using Linear.app.

Issue tab in GitHub I think was primarily supposed to be used for actual bugs, errors, warnings, etc 

I do not want to pollute the GitHub with more TODOs but we still need to track new features and TODOs. 

Another idea is that we can use a auto documentation package to make documentation from the comments.  But that would require more comments and install a package.